lib/util/user.js

```diff
@@ -1,4 +1,81 @@
+import clone from 'clone'
 
+/**
+ * Determines whether a loggedInUser is a new user
+ * that needs to complete the new account wizard.
+ */
 export function isNewUser (loggedInUser) {
   return !loggedInUser.hasConsentedToTerms
 }
+
+// Helper functions to determine if
+// a location is home or work.
+export const isHome = loc => loc.type === 'home'
+export const isWork = loc => loc.type === 'work'
+export const isHomeOrWork = loc => isHome(loc) || isWork(loc)
+
+/**
+ * An index of common place types, used to
+ * obtain icons for place types and to initialize place defaults.
+ */
+export const PLACE_TYPES = {
+  custom: {
+    icon: 'map-marker',
+    name: 'Custom',
+    type: 'custom'
+  },
+  dining: {
+    icon: 'cutlery',
+    name: 'Dining',
+    type: 'dining'
+  },
+  home: {
+    icon: 'home',
+    name: 'Home',
+    type: 'home'
+  },
+  work: {
+    icon: 'briefcase',
+    name: 'Work',
+    type: 'work'
+  }
+  // TODO: add more non-home/work types
```

Comment:
I don't think this comment adds much value and could be deleted.

Reply:
I updated the comment for this variable in https://github.com/opentripplanner/otp-react-redux/pull/280/commits/53797da17e0d91c1c07659bbd169f4e14ec8683f.